test/fgpu/test_pfb.py

```diff
@@ -56,13 +57,14 @@ def test_pfb_fir(context: AbstractContext, command_queue: AbstractCommandQueue,
     rng = np.random.default_rng(seed=1)
     h_in = rng.integers(0, 256, samples * SAMPLE_BITS // BYTE_BITS, np.uint8)
     weights = rng.uniform(-1.0, 1.0, (2 * channels * taps,)).astype(np.float32)
-    expected = pfb_fir_host(h_in, channels, unzip_factor, weights)
+    expected, expected_total_power = pfb_fir_host(h_in, channels, unzip_factor, weights)
```

Comment:
Not necessarily worth an update/further commit, but I would've liked to see an equivalent suffix update to the original `expected` - similar to how you've done `h_{out, total_power}`.

Reply:
Done - I like how that looks.